src/components/Atoms/TextInputWithDropdown/TextInputWithDropdown.js

```diff
@@ -178,7 +176,6 @@ const Options = ({
 };
 
 TextInputWithDropdown.propTypes = {
-  value: PropTypes.string.isRequired,
```

Comment:
Aha ok cool; it may be related to this: https://github.com/comicrelief/component-library/pull/564/files

I guess I'd just ask you to doublecheck this CL update within CRcom to make sure it doesn't introduce any problems into any context that is using the Typeahead (it may just be the Parkrun FSU)